source/webconfig/wifi_easymesh_translator.c

```diff
@@ -354,7 +475,7 @@ webconfig_error_t   translate_radio_object_to_easymesh_for_dml(webconfig_subdoc_
         }
         strncpy(em_radio_info->id.name,radio_iface_map->radio_name, sizeof(em_interface_name_t));
         mac_address_from_name(radio_iface_map->interface_name, em_radio_info->id.mac);
-        for (unsigned int j = 0 ; j < oper_param->numOperatingClasses; j++) {
+        for (unsigned int j = 0 ; j < oper_param->numOperatingClasses ; j++) {
```

Comment:
space should not be there after j=0 and semicolon (';'), same thing applies for numOperatingClasses and ';'